### PR TITLE
Azure/AWS deployment update

### PR DESCRIPTION
**Pull request by matkuliak**
## Summary of the changes / Why this is an improvement

- Update of deployment from AWS/Azure marketplaces 
- Slight reorganization of the section

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/cluster-deployment/deploy-to-cluster-marketplace/deploy-to-cluster-aws/subscribe-aws.rst [hunk 1]
```diff
@@ -56,7 +55,10 @@ configuration on the CrateDB Cloud end.
 .. image:: ../../../_assets/img/aws-subscribe-confirm.png
    :alt: CrateDB Cloud on AWS subscription confirmation
 
-Click on the *Set Up Your Account* button to proceed.
+After clicking the *Set Up Your Account*, you will be redirected to the CrateDB
+Cloud console. From here, you can follow the :ref:`usual deployment procedure
+<cluster-deployment-stripe>` use your new AWS subscription for the billing in
+the CrateDB Cloud console.
```
Comment (SStorm) [inline docs/cluster-deployment/deploy-to-cluster-marketplace/deploy-to-cluster-aws/subscribe-aws.rst:61]: Perhaps worth adding a note that AWS Subscriptions can take up to 10 minutes to be confirmed, before they are usable in CrateDB Cloud.

**Review by SStorm — APPROVED** [review]

File: docs/cluster-deployment/deploy-to-cluster-marketplace/deploy-to-cluster-aws/subscribe-aws.rst (hunk 1 above)
Comment (matkuliak) [reply docs/cluster-deployment/deploy-to-cluster-marketplace/deploy-to-cluster-aws/subscribe-aws.rst:61]: Oh, didn't notice that, mine was ready in a couple of seconds. Will add, thanks! 
Is that the case also for Azure? @SStorm

Comment (SStorm) [reply docs/cluster-deployment/deploy-to-cluster-marketplace/deploy-to-cluster-aws/subscribe-aws.rst:61]: No, Azure is usually a few seconds, AWS can take a bit longer.

**Review by kojinkai — APPROVED** [review]
Looks good to me

**Pull request merged**